docs/developer/releasing.md

```diff
@@ -94,17 +94,28 @@ For a new release branch, the release shepard will:
    fixes are available, cherry-picking the fixes into the release branch and
    starting a new release candidate.
 
-7. Create a PR to update code for the upcoming stable release. The changelog
-   sections for the release candidates should be replaced with a single section
-   for the stable release.
+7. Create a PR to [update code](#updating-code) again - this time using the stable
+   release version. 
 
 8. Create the stable release tag.
 
-9. Force-push the `release` branch to point at the stable release tag. This
+9. Make sure [Homebrew is updated](#updating-homebrew).
```

Comment:
The order here seems wrong. Updating Homebrew can only happen after the release is published.